x-pack/plugins/security_solution/server/integration_tests/lib/helpers.ts

```diff
@@ -54,9 +56,6 @@ export async function setupTestServers(logFilePath: string, settings = {}) {
   const root = createRootWithCorePlugins(
     deepmerge(
       {
-        server: {
```

Comment:
yes, at first, I added this config to have a "known" port where the Kibana instance was listening, just for debugging purposes, but without this config, it searches a opened port, which IMHO it's better to avoid any possible port collisions